### DISCUSSION TITLE
is there a way to customize layers in OSM preview mode ?

### DISCUSSION THREAD

**Discussion by julcnx** [openstreetmap/iD#9003] [Q&A] [answered]
Not sure if this is related to iD, but it would be great for sharing purposes to be able to customize the layer in read-only mode.

Example: https://www.openstreetmap.org/way/1031096561/#layers=Y

In the layers menu, only a fixed list of renderers is made available.

It would be great if the selection could be customizable at least through a query or hash parameter, as the editor does.

**Comment by tordans** [comment, marked as answer]
@cmoffroad the URL you shared is for https://github.com/openstreetmap/openstreetmap-website.
